ldes-server-infra-postgres/postgres-fragmentation-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/postgres/entity/MemberBucketisationEntity.java

```diff
@@ -0,0 +1,56 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.postgres.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.postgres.entity.MemberBucketisationEntity.BUCKETISATION;
+
+@Entity
+@Table(name = BUCKETISATION, indexes = {
+        @Index(columnList = "viewName"),
+        @Index(columnList = "fragmentId"),
+        @Index(columnList = "sequenceNr")
+})
+public class MemberBucketisationEntity {
+    public static final String BUCKETISATION = "bucketisation";
```

Comment:
can you prefix this ? fragmentation_bucket

Reply:
also in general, i'd call in memberBucketEntity since otherwise it sounds too much like the process instead of a table